crates/bevy_scene/src/lib.rs

```diff
@@ -22,11 +22,32 @@ use bevy_app::prelude::*;
 use bevy_asset::AddAsset;
 use bevy_ecs::{schedule::ExclusiveSystemDescriptorCoercion, system::IntoExclusiveSystem};
 
+/// Bevy scene saving/loading [`Plugin`].
+///
+/// # Contents
+///
+/// - Assets
+///
+///   [`DynamicScene`], [`Scene`]
+///
+/// - AssetLoaders
+///
+///   [`SceneLoader`]
+///
+/// - Resources
+///
+///   [`SceneSpawner`]
+///
+/// - Systems
+///
+///   - [`CoreStage::PreUpdate`]
+///     - <code>[scene_spawner_system].[exclusive_system](IntoExclusiveSystem::exclusive_system)().[at_end](ExclusiveSystemDescriptorCoercion::at_end)()</code>
```

Comment:
This table of contents pretty much duplicates the code in `<ScenePlugin as Plugin>::build`. The reason I still added it here is that you can't easily navigate to definitions in the online docs.

Making sure this list stays up to date may be too much of a hassle right now, though. Let me know if I should remove it (in favour of a note to check the source code).